### PR TITLE
Notice banner

### PR DESCRIPTION
**Pull request by panosvoudouris**
Notice banner implementation, see the Notice Banner component story.

### CODE REVIEW

File: styleguide/components/notice-banner.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import wrapper from './component-wrapper';
+
+const noticeBanner = () => {
+    const content = text(
+        'Banner content',
+        'If you’re a Thomas Cook customer and you’re stuck abroad or want to get your money back, get help from the Civil Aviation Authority.'
+    );
+
+    const component = `<div class="cads-notice-banner">
+    <h4>Notice</h4>
```
Comment (soniaturcotte) [inline styleguide/components/notice-banner.js]: I think this needs to be a span and not a heading. Does it make sense to have `<span class="h4">`

Comment (panosvoudouris) [reply styleguide/components/notice-banner.js]: Depends how it would be read out, if span is more appropriate we can do the span with h4 style.

Comment (panosvoudouris) [reply styleguide/components/notice-banner.js]: Also, the large version is just for the large breakpoint or md/lg?

File: scss/6-components/_notice-banner.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
+.cads-notice-banner {
+    @include cads-border(
+        l,
+        $cads-border-width-large,
+        0,
+        $cads-language__focus-colour
```
Comment (soniaturcotte) [inline scss/6-components/_notice-banner.scss]: should be `$cads-language__brand-secondary` otherwise, perfect 👍

Comment (panosvoudouris) [reply scss/6-components/_notice-banner.scss]: Yes it should!

**Pull request merged**